### DISCUSSION TITLE
Property selection of Web Api project

### DISCUSSION THREAD

**Discussion by NitaiBanik** [OpenCodeFoundation/eSchool#377] [Q&A] [answered]
I am confused while trying to add a WebApi project.
![image](https://user-images.githubusercontent.com/39862861/115122819-ccf20780-9fdb-11eb-98ed-d1f220b0649d.png)
Which options should I select?

Thanks in advance.

//cc: @mahedee @ratanparai @aftabkajal

**Comment by ratanparai** [comment, marked as answer]
@NitaiBanik: You do not need `HTTPS` and automatically created `Docker` file configured. `HTTPS` is not in our timeline now and we need to create custom `Dockerfile` for our services. 

I prefer `dotnet` cli to create new project. It is much easier to keep the folder structure in `Solution` level when using the `dotnet` cli than `Visual Studio`. 

If you want to use `Visual Studio` then, you have to create folders from the file `Explorer` and then also create `Solution folder` from `Visual Studio` in the same structure as your explorer's folder structure. I am not expert in `Visual Studio`, can not help you much regarding this. 

In summary, Untick both `Configure for HTTPS` and `Enable Docker`.
